cypress/support/fragments/inventory/inventoryActions.js

```diff
@@ -6,6 +6,12 @@ const importButtonInModal = Button('Import');
 const OCLWorldCatIdentifierTextField = TextField('Enter OCLC WorldCat identifier');
 const importTypeSelect = Select({ name :'externalIdentifierType' });
 
+function verifyFileNameDate(actualDate) {
+  expect(actualDate).to.be.greaterThan(Date.now() - 100000);
```

Comment:
keep 100sec in const. Can it be calculated through sys var downloadTimeout ?

Reply:
I will take less value instead of 100, but now `"downloadTimeout": 1000` It is good for waiting download but it too small for difference inside file name date. In file name

Reply:
downloadTimeout*10?

Reply:
In this test I can get accurate time for check time in filename because it may have differences 10-20 sec. It not depends of download process. It depends of what time was when file start generated. I can't track this time with accurate to the second so I need some interval. 
In this case we don't need `downloadTimeout` because this timeout needs for waiting download files, not for time when file starts to generate

Reply:
let's add local const with and specify value based on your experience. Have you seen time 100 sec in your runs?